tests/Backend/ExternalBuckets/SnowflakeExternalBucketShareTest.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Test\Backend\ExternalBuckets;
+
+use Exception;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\Workspaces;
+use Keboola\TableBackendUtils\Escaping\Snowflake\SnowflakeQuote;
+use Keboola\Test\Backend\WorkspaceConnectionTrait;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+use Keboola\Test\Utils\ConnectionUtils;
+use const STORAGE_API_LINKING_TOKEN;
+
+class SnowflakeExternalBucketShareTest extends BaseExternalBuckets
+{
+    use WorkspaceConnectionTrait;
+    use ConnectionUtils;
+
+    public const EXTERNAL_DB = 'EXT_DB';
+    public const EXTERNAL_SCHEMA = 'EXT_SCHEMA';
+    public const EXTERNAL_TABLE = 'EXT_TABLE';
+
+    /** @var Client */
+    protected $shareClient;
+
+    /**
+     * @var Client
+     */
+    protected $linkingClient;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->initEmptyTestBucketsForParallelTests();
+        $this->shareClient = $this->getClientForToken(
+            STORAGE_API_SHARE_TOKEN,
+        );
+        $this->linkingClient = $this->getClientForToken(
+            STORAGE_API_LINKING_TOKEN,
+        );
+    }
+
+    public function testShareExternalBucket(): void
+    {
+        $STAGE = 'in';
```

Comment:
velke pismena pouzivame len na const ale asi to nicomu nevadi :)

Reply:
je to vlastně konstanta pro daný test, není pro celou třídu, nějak mě nenapadlo, jak jinak to lépe napsat

Návrhy?

Reply:
jasne vram je to asi jedno za mna to moze byt kludne `$stage`

Reply:
ok